x/relationships/client/cli/tx.go

```diff
@@ -51,7 +51,7 @@ func GetCmdCreateRelationship(cdc *codec.Codec) *cobra.Command {
 				return err
 			}
 
-			msg := types.NewMsgCreateRelationship(cliCtx.FromAddress, receiver)
+			msg := types.NewMsgCreateRelationship(cliCtx.FromAddress, receiver, args[1])
```

Comment:
Should we check the validity of the subspace directly here since we can? I think this might make the command fail earlier which is what we should always do. What do you think? :thinking:

Reply:
Yeah sure, we can leave the check both on message (since they are reached if someone use REST cmds) and inside the CLI